Sources/XiCliCore/CommandLineTool.swift

```diff
@@ -27,19 +27,24 @@ public final class CommandLineTool {
             return
         }
         
-        guard let fileInput = args.fileInput else {
+        if args.fileInputs.isEmpty {
```

Comment:
```suggestion
        guard !args.fileInputs.isEmpty else {
```
This is not necessary, but it would be a nice addition to convey that it should always return.